main.go

```diff
@@ -98,7 +105,11 @@ func main() {
 		}
 		defer w.Close()
 	}
-	fmt.Fprintf(w, header, *flagPkg)
+	headerText, err := header(*flagPkg, !(*flagPrivate))
+	if nil != err {
+		log.Fatalf("Failed to expand autogenerated code: %v", err)
+	}
+	fmt.Fprint(w, headerText)
```

Comment:
With the `[]byte` return below, you can do just `w.Write(headerText)`.

And I know it wasn't done before, but could you add error checking to that as well?

```
if _, err := w.Write(headerText); err != nil {
  log.Fatalf("failed to write output: %s", err)
}
```